### ISSUE TITLE
Preact please

### ISSUE THREAD

**Issue by everythinginjs** [nxext/nx-extensions#1100]
Hello @DominikPieper thanks for great nx-extentions, 
I am looking a way to add Preact to my NX workspace. 
I can see nxext/preact but deprecated!

**Comment by everythinginjs** [comment]
Okay I see there is a preact already added but there are some dependency tree conflicts which I need to install with --legacy-peers-deps flag

**Comment by DominikPieper** [comment]
@everythinginjs  yes there was a preact plugin, but I deprecated it cause there were not that many users. Some time ago I minimized the number of plugins, removing the ones slightly used

**Comment by everythinginjs** [comment]
Thanks for the reply, I just started using NX not professional yet, but I do not think bumping up packages takes that much of time!, I also would like to use tailwind with it but seems to be not supported in this plugin.
